### ISSUE TITLE
Error syntax while it shouldn't due to ":" in value

### ISSUE THREAD

**Issue by stannie42** [adrienverge/yamllint#28]
Hi,

I am having the folowing error syntax while it shouldn't yield anything!

- msg: "blabla:42" > this ends in error syntax error: mapping values are not allowed here

Thanks,
Pierre.

**Comment by adrienverge** [comment]
Hi Pierre,

I can't reproduce the bug you describe:

```sh
$ cat >/tmp/test.yaml <<EOM
- msg: "blabla:42"
EOM
```
```sh
$ yamllint /tmp/test.yaml
/tmp/test.yaml
  1:1       warning  missing document start "---"  (document-start)
```
```sh
$ yamllint --version
yamllint 1.5.0
```

Could you paste the exact code snippet that is failing? Thanks!

**Comment by adrienverge** [comment]
Hey @stannie42, any news on that issue?

**Comment by varac** [comment]
@stannie42 I had the same error, turned out I had an old config `~/.config/yamllint/config` that I removed and the error was gone.

**Comment by adrienverge** [comment]
@oskarrough this is not the same issue.

Your YAML input seems incorrect, because `*` is a reserved keyword for YAML aliases (see [the YAML spec paragraph about that](https://yaml.org/spec/1.2/spec.html#id2786196)).

To have valid YAML, you can wrap the string inside quotes, or use a [folded-style string](https://yaml.org/spec/1.2/spec.html#id2760844), or a [literal-style string](https://yaml.org/spec/1.2/spec.html#id2795688).

**Comment by oskarrough** [comment]
Thank you, now I understand.

The use case is linting yaml in front-matter inside a markdown file. Issue is that `yamllint` is linting all of it as yaml, hence the error. I'll open a new issue.
